tests/test_on_data/behavior/test_behavior_interfaces.py

```diff
@@ -454,6 +454,40 @@ def check_read_nwb(self, nwbfile_path: str):
         pass
 
 
+@pytest.mark.skipif(
+    platform == "darwin" and python_version < version.parse("3.10"),
+    reason="interface not supported on macOS with Python < 3.10",
+)
+class TestDeepLabCutInterfaceFromCSV(DataInterfaceTestMixin):
```

Comment:
I was wondering why not to use the `DeepLabCutInterfaceMixin` but then I realize it is empty.

Reply:
Yeah, I was confused about that as well. Should we delete it? One of the testing classes uses it even though it does nothing...

Reply:
Yeah, I think we should delete it. Could you do the honors?